DiscordBeam/Base/src/main/java/de/lukweb/discordbeam/FileExtensionMarkdownLanguage.java

```diff
@@ -0,0 +1,27 @@
+package de.lukweb.discordbeam;
+
+public enum FileExtensionMarkdownLanguage {
+	KOTLIN("kt", "kotlin");
+
+	private final String extension;
+	private final String language;
+
+	FileExtensionMarkdownLanguage(String extension, String language) {
+		this.extension = extension;
+		this.language = language;
+	}
+
+	public static FileExtensionMarkdownLanguage getByExtension(String extension) {
+		for (FileExtensionMarkdownLanguage fileExtensionMarkdownLanguage : FileExtensionMarkdownLanguage.values()) {
+			if (fileExtensionMarkdownLanguage.extension.equals(extension)) {
```

Comment:
It's nothing important, but I would prefer a getter method for the `extension` attribute of the `FileExtensionMarkdownLanguage` enum.

Reply:
I decided to leave it out because it probably wouldn't be used other than for internal comparison, but I will add it 👍